### PR TITLE
Pass the adUnit array when emitting SET_TARGETING

### PR DESCRIPTION
**Pull request by ptomasroos**
We use this in our analytics adapter very useful to have as args

- [x] Feature

## Description of change

Since the auction process is detached from the DFP rendering process aka when does a call against DFP happen we find it very useful in our analytics adapter to have which ad unit's or all are getting invoked

### CODE REVIEW

File: src/prebid.js [hunk 1]
```diff
@@ -230,7 +230,7 @@ $$PREBID_GLOBAL$$.setTargetingForGPTAsync = function (adUnit) {
   targeting.setTargeting(targetingSet);
 
   // emit event
-  events.emit(SET_TARGETING);
+  events.emit(SET_TARGETING, adUnit);
```
Comment (mkendall07) [inline src/prebid.js]: adUnit is optional here. Probably better to define some structure. Is this an array, object etc?

Comment (mkendall07) [reply src/prebid.js]: Once that is done, it should be applied to the other SET_TARGETING emitter - see line 250

Comment (mkendall07) [reply src/prebid.js]: Just to be clear, my preference is:

events.emit(SET_TARGETING, { adUnits: [adUnit1, adUnitN...]});

Comment (ptomasroos) [reply src/prebid.js]: Thanks for the feedback. Will fix!

**Comment by mkendall07** [comment]
@ptomasroos 
any chance to work on this?

**Comment by ptomasroos** [comment]
Yes will fix tomorrow GMT time @mkendall07 forgot

**Comment by bretg** [comment]
@ptomasroos - any update?

**Comment by mkendall07** [comment]
closing as no update in several months.

**Pull request closed without merging**